### PR TITLE
TC 04.6.5_02<List of links with title “Gear’s deal” is on the left si…

### PR DESCRIPTION
**Pull request by MariaNikonova3101**
https://trello.com/c/iXKGJ8Py/432-tc-046502list-of-links-with-title-gears-deal-is-on-the-left-side

### CODE REVIEW

File: tests/menuSaleGearDeals.spec.js [hunk 1]
```diff
@@ -13,4 +13,14 @@ test.describe('menuSaleGear_Deals', () => {
         await expect(gearDealsTitleElement).toHaveText('Gear Deals');
 
     });
+
+    test('List of links with title “Gears deal” is on the left side', async ({ page }) => {
+        const itemsGearDeals = page.locator('.sidebar-main>.block-static-block.widget>.categories-menu>ul.items:last-child>li');
+        const GearDeals = page.locator('.categories-menu > strong:nth-child(5) > span')
+
+        await expect(itemsGearDeals).toHaveText(['Bags', 'Fitness Equipment']);
```
Comment (S-Lanchik) [inline tests/menuSaleGearDeals.spec.js:21]: Why you did not include 'Watches'?

Comment (MariaNikonova3101) [reply tests/menuSaleGearDeals.spec.js:21]: Sorry, but I don’t understand, could you clarify what you are talking about?

Comment (S-Lanchik) [reply tests/menuSaleGearDeals.spec.js:21]: Sorry for misunderstanding. I was looking on the other page where "Watches" also were included. Now I got what your test is checking.

**Review by S-Lanchik — APPROVED** [review]

**Comment by S-Lanchik** [comment]
Just add PR link to Trello.

**Review by alenavolakhava — APPROVED** [review]
hi @MariaNikonova3101 
your test passes, although what i would suggest is to update the name of your test as it's a little bit misleading. basically what your test is doing, it's verifying that the `Gear Deals` category contains the links `Bags` and `Fitness equipment `, but there is no way to verify that it's located on the left side of the page. we can only assert that an element exists on the page regardless of it's exact location
Screenshot 2024-04-22 at 8 44 49 PM


Screenshot 2024-04-23 at 9 25 08 AM

**Pull request merged**